test/e2e/utils/recordedit-helpers.js

```diff
@@ -321,7 +347,51 @@ exports.testPresentationAndBasicValidation = function(tableParams, isEditMode) {
                     });
                 });
             }
+            
+            if (jsonCols.length > 0) {
+                describe("JSON fields, ", function () {
+                    it("should show textarea input for JSON datatype and then set the value", function() {
+                        jsonCols.forEach(function(c) {
+                        chaisePage.recordEditPage.getTextAreaForAcolumn(c.name, recordIndex).then(function(jsonTxtArea) {
+                            if (jsonTxtArea) {
+                                expect(true).toBeDefined();
```

Comment:
This check is unnecessary. Instead of doing an `if / else` check around whether the value is defined or not, do this instead:
`expect(jsonTxtArea).toBeDefined()`
or a slightly more robust check would be:
`expect(jsonTxtArea).toEqual(jasmine.any(Object))`